src/services/bulksubmitstatus.service.ts

```diff
@@ -30,10 +30,10 @@ export async function bulkSubmitStatus(req: any, res: any) {
   const clientId = `${submitter.value}-${submissionId}`;
 
   logger.debug(`Retrieving submission status entry for client: ${clientId}`);
-  const submissionStatus = getBulkSubmissionStatus(submitter.value, submissionId);
+  const submissionStatus = await getBulkSubmissionStatus(submitter.value, submissionId);
 
   if (!submissionStatus) {
-    logger.debug(`Writing unable to find bulk import request OperationOutcome to file for client: ${clientId}`);
+    logger.debug(`Unable to find submission status OperationOutcome for client: ${clientId}`);
```

Comment:
I know this is what the error said before so not your fault, but is the submission status indeed an OperationOutcome? Would it be more accurate to just say "Unable to find submission status for client"?